test/resources/Invoices.spec.js

```diff
@@ -99,6 +99,20 @@ describe('Invoices Resource', function() {
       });
     });
 
+    describe('With a options object in addition to a customer ID', function() {
+      it('Sends the correct request', function() {
+
```

Comment:
oof, JSCS would yell at you if you put in these sorts of whitespaces in our JS! :P